### PR TITLE
Added python toolbox

### LINKED ISSUE #4: NC: Use python-toolbox
Use python-toolbox

* rename folder `noteboook_connector` to `exasol` with content `secret_store.py`
* see [bucketfs-python/pyproject.tom](https://github.com/exasol/bucketfs-python/blob/main/pyproject.toml):  `packages = [ {include = "exasol"}, ]`

### PR DESCRIPTION
**Pull request by ckunki**
closes #4

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (Nicoretti) [inline .github/workflows/ci.yml:1]: doesn't the toolbox provide a ci workflow?

File: doc/changes/changes_0.0.1.md [hunk 1]
```diff
@@ -7,3 +7,4 @@ This release adds the initial implementation of the secret store
 ## Changes
 
 * #1: Added secret store
+* #4: Used python toolbox
```
Comment (Nicoretti) [inline doc/changes/changes_0.0.1.md:10]:
```suggestion
* #4: Add python toolbox
```

Comment (ckunki) [reply doc/changes/changes_0.0.1.md:10]: AFAIK we agreed to always write in past tense in changes file. 
Would it be OK for you to write "added python toolbox"?

File: .github/workflows/ci.yml
Comment (ckunki) [reply .github/workflows/ci.yml:1]: As agreed:
* wokflows will come with next version of exasol python-toolbox.
* we will approve the current PR
* and update to new version of exasol python-toolbox incl. workflows in a separate PR asap

**Review by Nicoretti — APPROVED** [review]

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**